app/components/avo/views/resource_edit_component.html.erb

```diff
@@ -22,6 +22,24 @@
         icon: 'arrow-left' do %>
           <%= t('avo.cancel').capitalize %>
         <% end %>
+        <% #TODO: for some reason new button act as delete button when delete button is present %>
```

Comment:
@adrianthedev I'm having some troubles here, when i put `method: :delete` on destroy button, both, save and destroy buttons acts as destroy button, if I remove `method: :delete`, both buttons acts like edit. I don't know how to make 2 separate actions for the same form without some kind of param and in the routes check if param is present to decide if we go to update or to destroy route.

Reply:
Maybe a data action and an AJAX request solve this, i don't know what is the best approach

Reply:
actually changing it from button to link solved this issue